app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Here you can try to add a default description, so you don't have to wright the same descriptions always